src/options/flags.rs

```diff
@@ -35,6 +35,8 @@ pub static GIT_IGNORE:  Arg = Arg { short: None, long: "git-ignore",           t
 pub static DIRS_FIRST:  Arg = Arg { short: None, long: "group-directories-first",  takes_value: TakesValue::Forbidden };
 pub static ONLY_DIRS:   Arg = Arg { short: Some(b'D'), long: "only-dirs", takes_value: TakesValue::Forbidden };
 pub static ONLY_FILES:   Arg = Arg { short: Some(b'f'), long: "only-files", takes_value: TakesValue::Forbidden };
+pub static ONLY_LINKS:   Arg = Arg { short: None, long: "only-links", takes_value: TakesValue::Forbidden };
+pub static NO_LINKS:   Arg = Arg { short: None, long: "no-links", takes_value: TakesValue::Forbidden };
```

Comment:
This part should be aligned like the lines above (could you also realign line 37?)